EarlGrey.xcodeproj/project.pbxproj

```diff
@@ -973,6 +991,8 @@
 				OTHER_CFLAGS = (
 					"-fobjc-arc",
 					"-fobjc-arc-exceptions",
+					"-DNS_BLOCK_ASSERTIONS=1",
```

Comment:
This line is not required. No reason to define it, just to turn around and undefine it.